### PR TITLE
Hide item options (share, overflow menu) & overflow items in move/copy dialog

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Before & after

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java [hunk 1]
```diff
@@ -102,13 +102,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Action bar setup
         setupToolbar();
-        getSupportActionBar().setDisplayShowTitleEnabled(true);
+
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().setDisplayShowTitleEnabled(true);
+            getSupportActionBar().setTitle(caption);
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java]: The caption gets dynamically calculated in the lines below ;)

**Comment by AndyScherzinger** [comment]
👍 Just a minor comment 👍

File: src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java]: Right. I changed the order 👍

**Comment by AndyScherzinger** [comment]
pinging @mario for second review :)

**Comment by mario** [comment]
👍

**Pull request merged**